services/ui-src/src/components/widgets/SpreadsheetWidget.tsx

```diff
@@ -4,14 +4,14 @@ import { Box, Flex, Image, Text } from "@chakra-ui/react";
 // assets
 import greenSpreadsheetIcon from "../../assets/icons/icon_spreadsheet_green.png";
 
-export const SpreadsheetWidget = ({ description, ...props }: Props) => {
+export const SpreadsheetWidget = ({ description, alt, ...props }: Props) => {
   return (
     <Box {...props}>
       <Flex sx={sx.container}>
         <Flex sx={sx.iconContainer}>
           <Image
             src={greenSpreadsheetIcon}
-            alt="Excel Workbook Icon"
+            alt={alt != undefined ? alt : "Excel Workbook Icon"}
```

Comment:
Nit: Could use nullish coalescing here:
```suggestion
            alt={alt ?? "Excel Workbook Icon"}
```